tests/unit/store/utility.spec.ts

```diff
@@ -0,0 +1,282 @@
+        createDummyAccentPhrase(["き", "け"]), // 最初・真ん中・最後を消去
+        createDummyAccentPhrase(["た", "ち", "つ", "て", "と"]), // すべて置き換え
+      ];
+      const tuningTransctiption = new TuningTranscription(before, after);
+      const result = tuningTransctiption.transcribe();
+
+      // モーラ数などは変わっていない
+      expect(result.length).toEqual(after.length);
+      for (let i = 0; i < result.length; i++) {
+        expect(result[i].moras.length).toEqual(after[i].moras.length);
+      }
+
+      // 転写されている
+      ["い", "え", "き", "け"].forEach((moraText) => {
+        expect(findMora(result, moraText)).toEqual(findMora(before, moraText));
+      });
+
+      // 転写されていない
+      ["あ", "う", "お", "た", "ち", "つ", "て", "と"].forEach((moraText) => {
+        expect(findMora(result, moraText)).not.toEqual(
+          findMora(before, moraText)
+        );
+      });
+    });
+
+    it("拗音のあるモーラも正しくデータを転写できる", async () => {
+      const before = [
+        createDummyAccentPhrase(["い", "しぃ", "う", "しゅ", "お", "しょ"]),
+      ];
+      const after = [
+        createDummyAccentPhrase(["あ", "しゃ", "き", "きゅ", "お", "しょ"]),
+      ];
+      const tuningTransctiption = new TuningTranscription(before, after);
+      const result = tuningTransctiption.transcribe();
+
+      // モーラ数などは変わっていない
+      expect(result.length).toEqual(after.length);
+      for (let i = 0; i < result.length; i++) {
+        expect(result[i].moras.length).toEqual(after[i].moras.length);
+      }
+
+      // 転写されている
+      ["お", "しょ"].forEach((moraText) => {
+        expect(findMora(result, moraText)).toEqual(findMora(before, moraText));
+      });
+
+      // 転写されていない
+      ["あ", "しゃ", "き", "きゅ"].forEach((moraText) => {
+        expect(findMora(result, moraText)).not.toEqual(
+          findMora(before, moraText)
+        );
+      });
+    });
+  });
+
+  describe("isAccentPhrasesTextDifferent", () => {
+    it("アクセントフレーズのテキストが異なる場合、trueを返す", () => {
+      const accentPhrases1 = [
+        createDummyAccentPhrase(["あ", "い", "う"]),
+        createDummyAccentPhrase(["え", "お"]),
+      ];
+      const accentPhrases2 = [
+        createDummyAccentPhrase(["か", "き", "く"]),
+        createDummyAccentPhrase(["け", "こ"]),
+      ];
+      const result = isAccentPhrasesTextDifferent(
+        accentPhrases1,
+        accentPhrases2
+      );
+      expect(result).toBe(true);
+    });
+
+    it("アクセントフレーズのテキストが同じ場合、falseを返す", () => {
+      const accentPhrases1 = [
+        createDummyAccentPhrase(["あ", "い", "う"]),
+        createDummyAccentPhrase(["え", "お"]),
+      ];
+      const accentPhrases2 = [
+        createDummyAccentPhrase(["あ", "い", "う"]),
+        createDummyAccentPhrase(["え", "お"]),
+      ];
+      const result = isAccentPhrasesTextDifferent(
+        accentPhrases1,
+        accentPhrases2
+      );
+      expect(result).toBe(false);
+    });
+  });
+
+  test("buildAudioFileNameFromRawData", () => {
+    const fileNamePattern = "テストパターン";
```

Comment:
もうちょっとテストの網羅性を上げられるかなと思ったのでコメントさせていただきます！
```suggestion
    const fileNamePattern = "テストパターン";  // FIXME: いろんなパターンを試す
```